openshift/postgresql.dc.json

```diff
@@ -172,6 +186,25 @@
         "oracle-fdw-schema": null
       }
     },
+    {
```

Comment:
You can probably move this to the python deployment config until you get a separate postgres deployment config